cmd/istioctl/main.go

```diff
@@ -467,3 +495,55 @@ func newClient() (*crd.Client, error) {
 		model.DestinationPolicy,
 	}, "")
 }
+
+func preprocMixerConfig(configs []crd.IstioKind) error {
+	for i, config := range configs {
+		if config.Namespace == "" {
+			configs[i].Namespace = namespace
+		}
```

Comment:
Also set `apiGroup` if it is not specified.

Reply:
done